addons/website/static/src/js/website.ace.js

```diff
@@ -128,6 +133,7 @@ var ViewEditor = Widget.extend({
         }
         document.body.addEventListener('mouseup', stopResizing, true);
         self.$('.ace_gutter').mouseup(stopResizing).mousedown(startResizing).click(stopResizing);
+        self.$('.ace_content').mouseup(stopResizing).click(stopResizing);
```

Comment:
I use stopResizing method in `.ace_content` to solve this issue.
https://www.youtube.com/watch?v=Lngsm3ytT4w